compare/benches/bench_bits.rs

```diff
@@ -699,14 +699,48 @@ fn random_bit_reads(c: &mut Criterion) {
     group.finish();
 }
 
+fn write_bytes(c: &mut Criterion) {
+    let data = gen_data();
+
+    let mut group = c.benchmark_group("write_bytes");
+
+
+    for i in &[4, 8, 16, 80, 240, 960] {
+        group.throughput(Throughput::Bytes((*i) as u64));
+
+        let data = data[..*i].to_vec();
+
+        group.bench_with_input(BenchmarkId::new("aligned", i), &i, |b, param| {
+            let mut buf = vec![0u8; **param];
+            b.iter(|| {
+                let mut bitter = LittleEndianWriter::new(&mut buf);
+                bitter.write_bytes(&data)
+            })
+        });
+
+        group.bench_with_input(BenchmarkId::new("unaligned", i), &i, |b, param| {
+            let mut buf = vec![0u8; **param + 1]; // To fit the extra bit and prevent vec extension
```

Comment:
I think if we pull out the write_bit out of the loop iterator, it better focuses the write_bytes:

```rust
            let mut bitter = LittleEndianWriter::new(std::io::sink());
            bitter.write_bit(true).expect("should write bit");
            b.iter(|| {
                bitter.write_bytes(&data)
            })
```